### PR TITLE
fix: better mime type support for generic types

### PR DESCRIPTION
**Pull request by juliusmarminge**
...

### CODE REVIEW

File: packages/shared/src/component-utils.ts [hunk 1]
```diff
@@ -2,21 +2,25 @@ import type { CSSProperties, ReactNode } from "react";
...
+    return [
+      // Add wildcard to support all subtypes, e.g. image => "image/*"
+      `${type}/*`,
+      // But some browsers/OSes don't support it, so we'll also dump all the mime types
+      // we know that starts with the type, e.g. image => "image/png, image/jpeg, ..."
+      ...objectKeys(mimeTypes).filter((t) => t.startsWith(type)),
+    ].join(", ");
   });
-
-  if (accepted.includes("blob")) {
```
Comment (markflorkowski) [inline packages/shared/src/component-utils.ts:16]: I think this case is still needed?

Comment (markflorkowski) [reply packages/shared/src/component-utils.ts:16]: nvm, I see it moved up, missed that at first

Comment (juliusmarminge) [reply packages/shared/src/component-utils.ts:16]: i early-returned this case first line in the function

**Review by markflorkowski — APPROVED** [review]
Why did tests break...?

**Comment by juliusmarminge** [comment]
hmm passes locally for me 🤔

**Pull request merged**